modules/apps/password-policies-admin/password-policies-admin-web/src/main/resources/META-INF/resources/js/EditPasswordPolicyAssignmentsPropsTransformer.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {createRenderURL, openSelectionModal} from 'frontend-js-web';
+
+import {ACTIONS} from './actions';
+
+export default function propsTransformer({
+	additionalProps,
+	portletNamespace,
+	...props
+}) {
+	return {
+		...props,
+		onActionButtonClick(event, {item}) {
+			const action = item.data?.action;
+
+			if (action) {
+				event.preventDefault();
```

Comment:
If we use `<ClayButton>`, we don't need `preventDefault`. It has `type="button"`, preventing form submission.

Reply:
:+1

Reply:
Changed [here](https://github.com/drewbrokke/liferay-portal/blob/c2ae813b8ed2cab963db8265552bab1428850271/modules/apps/password-policies-admin/password-policies-admin-web/src/main/resources/META-INF/resources/js/EditPasswordPolicyAssignmentsManagementToolbarPropsTransformer.js#L81-L83)